README.md

```diff
@@ -320,6 +323,28 @@ object that may contain any of the following fields:
   * autosettle - Whether received messages should be automatically
     settled once the remote settles them. Defaults to true.
 
+And attach frame fields:
+
+  * role
```

Comment:
The role cannot be set.

Reply:
Ah. True. I copied them from code and dropped some, but this was left.

Reply:
The incomplete_unsettled and initial_delivery_count are also internal.